src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXLegalityCheck.cpp

```diff
@@ -250,6 +278,10 @@ bool isSuitableForZDNN(OP_TYPE op, const DimAnalysis *dimAnalysis) {
 template <>
 bool isSuitableForZDNN<ONNXAddOp>(
     ONNXAddOp op, const DimAnalysis *dimAnalysis) {
+  // Check NNPA level.
+  if (!isCompatibleWithNNPALevel(NNPA_Z16)) {
+    return false;
+  }
```

Comment:
Could you please remove `{` and `}` for this simple if? Same for other locations.

Reply:
@tungld unless it's a standard in onnx-mlir to drop the `{}`'s for one line `if` statements, I lean towards keeping them. Functionally nothing changes but the braces are less error prone during maintenance or debugging scenarios.

Reply:
@cjvolzka we don't have a standard but we follow LLVM coding style: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

Reply:
Ok, good to know. I defer to the established norm for the repo on such things.

Reply:
Thanks, updated to remove { } from these statements.

Reply:
The only places I think it's ok to have extra `{}` is in the presence of nested ifs, as dangling `else` are dangerous. Otherwise, best to keep to the LLVM formatting rules